tests/data_validation/data_validation_list_component.test.ts

```diff
@@ -233,6 +238,33 @@ describe("autocomplete in composer", () => {
       expect(getCellContent(model, "A1")).toBe("hello");
       expect(model.getters.getEditionMode()).toBe("inactive");
     });
+
+    test("Dropdown lists are kept when clearing the content of a non-empty cell", () => {
```

Comment:
those tests could be move elsewhere than this file (component tests). I guess there are already other plugin tests for lists

Reply:
While there were a plugin for the checkbox, I don't think there is any plugin tests for dropdown list, I'll create one :)